src/components/Input/Input.jsx

```diff
@@ -0,0 +1,178 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+import { v4 as uuid } from 'uuid';
+import Cleave from 'cleave.js/react';
+import * as InputMasks from './InputMasks';
+import './Input.scss';
+
+const Input = ({
+  autoComplete,
+  autoFocus,
+  className,
+  id,
+  inputMask,
+  isDisabled,
+  isRequired,
+  label,
+  name,
+  onBlur,
+  onChange,
+  onFocus,
+  placeholder,
+  type,
+  value,
+}) => {
+
+  const handleChange = (e) => {
```

Comment:
Think I'm for writing out the word event; it's a little longer, but more obvious. 

The linter error on this line is because only a single argument is being passed to this function, yet it's wrapped in parenthesis. So, this function could look like:

```
const handleChange = event => {
  onChange(e);
};
```

Reply:
Hah I like to too actually. I'm generally a fan of long variable names. It got flagged due to it being an IE global I think. Let's come back to this. Maybe we can just disable this in the linter.